server/test/integration/routes/projects.spec.js

```diff
@@ -186,6 +186,129 @@ describe('integration: projects', () => {
         });
     });
   });
+
+  describe('/:id PUT', () => {
+    const project = {
+      name: 'Self-overwriting-project',
+      category: 'Hardware',
+      users: [
+        {
+          firstName: 'replaceable',
+          lastName: 'Kid',
+          dob: '2008-12-01T20:00.000Z',
+          gender: 'male',
+          type: 'member',
+        },
+        {
+          firstName: 'exchangeable',
+          lastName: 'orga',
+          dob: '1991-12-01T20:00.000Z',
+          gender: 'male',
+          email: 'updatable@example.com',
+          phone: '3538123123123',
+          country: 'IE',
+          type: 'supervisor',
+        },
+      ],
+    };
+    let refProject;
+    let _token;
+
+    before(() => {
+      return util.user.create('updater@example.com')
+        .then((ntoken) => {
+          _token = ntoken;
+          return util.project.create(_token, eventId, project);
+        })
+        .then((res) => {
+          return util.project.get(_token, eventId, res.body.id);
+        })
+        .then((res) => { refProject = res.body; });
+    });
+
+    it('should update a project (same struct)', async () => {
+      const payload = project;
```

Comment:
Should be cloning? The below changes will cause `project` to be updated and affect future tests.